dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/info/NetworkInfoDeserializer.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.internal.net.info
+
+import com.datadog.android.core.internal.domain.Deserializer
+import com.datadog.android.core.internal.utils.sdkLogger
+import com.google.gson.JsonParseException
+import com.google.gson.JsonParser
+
+internal class NetworkInfoDeserializer : Deserializer<NetworkInfo> {
```

Comment:
Do you still need this class ?

Reply:
The next step will be to switch to the newly generated NetworInfo. I will decide then if I will need it or not...maybe I will still keep it to avoid calling those static methods directly and to keep a composable architecture.